tests/e2e/testhelpers/environment.go

```diff
@@ -78,7 +78,7 @@ func CreateEnvironmentRemote(org string, projectIdentifier string, environment,
 
 			return nil
 		},
-		retry.Attempts(5), retry.Delay(500*time.Millisecond),
+		retry.Attempts(5), retry.Delay(1000*time.Millisecond),
```

Comment:
Env creation in Prod seemed to be a little laggy this morning which was causing some other tests to fail so I've increased this retry to try and avoid flaky failures.